src/wallet/wallet.ts

```diff
@@ -180,6 +182,19 @@ export const generateLyraWallet = async (testnet: boolean, mnem: string): Promis
     return {mnemonic: mnem, xpub: hdwallet.derive(LYRA_DERIVATION_PATH).toJSON().xpub};
 };
 
+/**
+ * Generate ADA wallet
+ * @param mnemonic mnemonic seed to use
+ * @returns wallet
+ */
+ export const generateAdaWallet = async (mnemonic: string): Promise<Wallet> => {
+    const walletSecret = await cardanoCrypto.mnemonicToRootKeypair(mnemonic, 1);
```

Comment:
Use HD derivation path for Cardano from here -https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
Sorry for a very incomplete draft... I've walked down a standard Cardano derivation path now. 
Note that Cardano has used `ed25519` over `secp256k1` since Byron, hence switched the purpose field to `1852'` since Shelley to clearly distinguish this difference from the original BIP44 specification.